src/graph/validator/AdminJobValidator.cpp

```diff
@@ -49,6 +50,14 @@ Status AdminJobValidator::validateImpl() {
         }
       }
     }
+  } else if (reqOp == meta::cpp2::AdminJobOp::SHOW_All ||
```

Comment:
Here `stop `and `recover` job also need space.
Either the job under the current space, or the global job.
Cannot operate job under other space

Reply:
At present, nebula doesn't have any global job type. AFAIK, all commands about job should require space info.

Reply:
Please refer to  `requireSpace`() function in  AdminJobValidator.h